src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -559,9 +559,19 @@ var ReactDOM = {
 };
 
 if (typeof injectInternals === 'function') {
+  // A dummy function to check for minification during runtime
+  // Refer https://github.com/facebook/react-devtools/issues/694#issuecomment-300535376
+  var dummyMinificationTestFn = function() {
+    if (__DEV__) {
+      var anotherDummyNoop = function() {};
```

Comment:
How do we detect whether this code block was eliminated, judging by the source?
It can be tricky because it could be, for example, minified but not eliminated.

I think we should instead do something like `return 42;` and in `__DEV__` block instead.
Then, on DevTools side, we can check both that:

* The function didn’t return anything (DEV mode check)
* The function doesn’t contain 42 literal (dead code elimination check)